Config/StackBuilderInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Config;
+
+interface StackBuilderInterface
+{
+    /**
+     * @param string $filterSetName
+     * @param array  $filterSetData
+     *
+     * @return StackInterface
```

Comment:
this does not add anything over the phpdoc, lets remove it

Reply:
done